jsonschema-hyperschema.xml

```diff
@@ -506,18 +515,103 @@
                     <section title="Missing values">
                         <t>
                             Sometimes, the appropriate values will not be available.
-                            For example, the template might specify the use of object properties, but the instance is an array or a string.
+                            For example, the template might specify the use of object properties, but no such input was provide (or "hrefSchema" is not present), and the instance is an array or a string.
                         </t>
 
                         <t>
-                            If any of the values required for the template are not present in the JSON instance, then substitute values MAY be provided from another source (such as default values).
+                            If any of the values required for the template are present in neither the user input (if relevant) or the JSON instance, then substitute values MAY be provided from another source (such as default values).
                             Otherwise, the link definition SHOULD be considered not to apply to the instance.
                         </t>
                     </section>
                 </section>
 
             </section>
 
+            <section title="hrefSchema" anchor="hrefSchema">
+                <t>
+                    The value of the "hrefSchema" link description property MUST be
+                    a valid JSON Schema.  This schema is used to validate user input
+                    for filling out the URI Template in
+                    <xref target="href">"href"</xref>, as described in that section.
+                </t>
+                <t>
+                    Omitting "hrefSchema" or setting the entire schema to "false" prevents
+                    any user input from being accepted.
```

Comment:
@Relequestual I figured out why it made sense to me to do it this way.

Validation is a constraint system.  An empty schema validates everything, and every keyword you add restricts the set in some way.

Hyper-schema defines functionality.  So in my head thats an additive capability approach.  A blank hyper-schema indicates that there are known hypermedia transitions.  So to me, it's philosophically consistent to look at a link and say that a minimal link defines minimal functionality.  In order to know that it takes user input, it has to declare that it does.

I can see an argument in the other direction, though.  I just wanted to explain why it felt consistent to me.  I view it as a different conceptual model than validation.